boards/arm/mimxrt1062_fmurt6/mimxrt1062_fmurt6.dts

```diff
@@ -33,6 +33,9 @@
 		zephyr,console = &lpuart7;
 		zephyr,shell-uart = &lpuart7;
 		zephyr,canbus = &flexcan1;
+		zephyr,gps1 = &lpuart2;
```

Comment:
Are these well known Zephyr chosen node names? I don't see them being used anywhere in the tree. I would rather see these node names be devicetree aliases if these UART connections are being used in a downstream project

Reply:
Changed from Chosen to Aliases as suggested